docs/v1/P4Runtime-Spec.mdk

```diff
@@ -715,6 +717,9 @@ follows:
     * Non-OK (with `status.code` set to `google.rpc.ALREADY_EXISTS`) when the
       controller is determined to be a slave for a given (`device_id`,
       `role_id`).
+  An OK message is only sent after all previous write requests have been
```

Comment:
Carmelo and Jordan have confirmed that it would be useful to them to have a google.rpc.NOT_FOUND` Non-OK status sent to all the stand-bys when the master disconnects, so I would recommend adding this as part of the PR given the very small implementation cost
It may also provide some justification / value for the stand-by connections...

Reply:
@stefanheule I think it would be good to address this as you update the PR

Reply:
Done.